### DISCUSSION TITLE
Enabling / disabling documentation for selected sources

### DISCUSSION THREAD

**Discussion by vitreo12** [hrsh7th/nvim-cmp#1960] [Q&A]
Hello! 

Is it possible to selectively enable or disable documentation for certain source types? In my case, I would want to disable them for `tags` but enable them for `snippets`. Right now, I seem to only be able to enable or disable docs for all sources with:

```
cmp.setup {
      view = {
        docs = {
          auto_open = false,
        },
      },
}
```

**Comment by jonathan-elize** [comment]
I'm wondering the same thing. Did you ever figure something out?
